### PR TITLE
Add descrptions to rooms

### PR DESCRIPTION
**Pull request by netpro2k**
Adds descriptions to room settings and room info screens. Featured rooms on the homepage (favorites and public) with descriptions will get an info icon that can be clicked to bring up the room info dialog for that room.

Also fixes unrelated mismatched quote in schema.toml that is breaking the admin panel.

TODO
- [x] Merge https://github.com/mozilla/reticulum/pull/316

### CODE REVIEW

File: src/assets/stylesheets/shared.scss [hunk 1]
```diff
@@ -61,6 +61,12 @@ $favorited-color: var(--favorited-color, $duck-yellow);
   width: 100%;
 }
 
+%textarea {
```
Comment (gfodor) [inline src/assets/stylesheets/shared.scss]: this looks like a tag name to me, i'd rename. confused me

File: src/assets/translations.data.json [hunk 2]
```diff
@@ -46,6 +46,7 @@
     "sign-in.out": "Sign Out",
     "room-settings.apply": "Apply",
     "room-settings.name-subtitle": "Rooms Name",
+    "room-settings.description-subtitle": "Rooms Description",
```
Comment (gfodor) [inline src/assets/translations.data.json]: s/Rooms/Room for this and name

File: src/react-components/home-root.js [hunk 3]
```diff
@@ -104,6 +105,13 @@ class HomeRoot extends Component {
     this.showDialog(AuthDialog, { verifying, verified, authOrigin: this.props.authOrigin });
   };
 
+  showHubInfo = hubEntry => {
```
Comment (gfodor) [inline src/react-components/home-root.js]: I know these terms are getting mashed up, but probably `showRoomInfo` given the dialog name

File: src/react-components/media-tiles.js [hunk 4]
```diff
@@ -42,6 +43,7 @@ class MediaTiles extends Component {
     history: PropTypes.object,
     urlSource: PropTypes.string,
     handleEntryClicked: PropTypes.func,
+    handleEntryInfo: PropTypes.func,
```
Comment (gfodor) [inline src/react-components/media-tiles.js]: nit: `handleEntryInfoClicked`

File: src/react-components/media-tiles.js [hunk 5]
```diff
@@ -293,12 +302,18 @@ class MediaTiles extends Component {
                 </div>
               )}
             {isHub && (
-              <div className={styles.attribution}>
-                <div className={styles.lastJoined}>
-                  <FormattedMessage key="1" id="media-browser.hub.joined-prefix" />
-                  {dayjs(entry.last_activated_at).fromNow()}
+              <>
```
Comment (gfodor) [inline src/react-components/media-tiles.js]: this empty tag is a new one, is this intended? i've never seen that before.

File: src/react-components/media-tiles.js [hunk 6]
```diff
@@ -293,12 +302,18 @@ class MediaTiles extends Component {
...
+                <div className={styles.attribution}>
+                  <div className={styles.lastJoined}>
+                    <FormattedMessage key="1" id="media-browser.hub.joined-prefix" />
```
Comment (gfodor) [inline src/react-components/media-tiles.js]: is this `key` right?

File: src/react-components/room-info-dialog.js [hunk 7]
```diff
@@ -57,37 +62,50 @@ export default class RoomInfoDialog extends Component {
       );
     }
 
-    const title = <div className={styles.title}>{this.props.hubName}</div>;
+    const title = (
+      <div className={styles.title}>{hasDescription && hasScene ? "Room & Scene Info" : this.props.hubName}</div>
```
Comment (gfodor) [inline src/react-components/room-info-dialog.js]: translation

File: src/react-components/ui-root.js [hunk 8]
```diff
@@ -1729,13 +1731,15 @@ class UIRoot extends Component {
               stateKey="modal"
               stateValue="room_info"
               history={this.props.history}
-              render={() =>
-                this.renderDialog(RoomInfoDialog, {
+              render={props => {
+                console.log(props);
```
Comment (gfodor) [inline src/react-components/ui-root.js]: logging

File: src/react-components/media-tiles.js (hunk 5 above)
Comment (netpro2k) [reply src/react-components/media-tiles.js]: It is a "Fragment" https://reactjs.org/docs/fragments.html

File: src/react-components/media-tiles.js (hunk 6 above)
Comment (netpro2k) [reply src/react-components/media-tiles.js]: Nope, not sure how that got in there.

**Pull request merged**